internal/stream/stream.go

```diff
@@ -184,9 +187,38 @@ func (s *Stream) StartReader(reader Reader) {
 
 	sr.start()
 
-	for _, sm := range s.streamMedias {
+	for m, sm := range s.streamMedias {
 		for _, sf := range sm.formats {
 			sf.startReader(sr)
+			if m.Type == "video" {
+				cb := sf.runningReaders[sr]
+				if cb == nil {
+					continue
+				}
+
+				go func() {
+					for i, u := range s.CachedUnits {
+						var hasAU = false
+						switch tunit := u.(type) {
+						case *unit.H264:
+							hasAU = tunit.AU != nil
+						case *unit.H265:
+							hasAU = tunit.AU != nil
+						}
+						if hasAU {
+							cb(u)
```

Comment:
Looking at the callback call in `stream_format.go`, I think we also want to increment the bytesSent counter whenever this callback is called:

```go
atomic.AddUint64(s.bytesSent, size)
```

Reply:
done!